distro-build/opensuse.sh

```diff
@@ -29,7 +29,7 @@ bootstrap_distribution() {
 		mount --bind /dev "${WORKDIR}/opensuse-$(translate_arch "$arch")/dev"
 		mount --bind /proc "${WORKDIR}/opensuse-$(translate_arch "$arch")/proc"
 		mount --bind /sys "${WORKDIR}/opensuse-$(translate_arch "$arch")/sys"
-		chroot "${WORKDIR}/opensuse-$(translate_arch "$arch")" zypper dup --no-confirm
+		chroot "${WORKDIR}/opensuse-$(translate_arch "$arch")" zypper refresh && zypper dup --no-confirm
```

Comment:
You have to run `zypper refresh` as separate command as chroot will execute everything up to first `&&`.